kolibri/plugins/management/assets/src/vue/class-edit-page/index.vue

```diff
@@ -23,18 +23,17 @@
     </div>
 
     <div class="toolbar">
-      <div class="searchbar" role="search">
+      <div class="searchbar" :class="{'searchbar-mobile': elSize.width < 600}" role="search">
```

Comment:
we should probably chat about this and you can show me what you mean - I don't understand